modules/ROOT/partials/incpg-icr-conflict.adoc

```diff
@@ -27,11 +27,11 @@ Not surprising since in both instances Couchbase Lite is acting like the active
 [NOTE]
 --
 Conflicts are only resolved during a *pull* replication.
-If conflicts are likely, you should configure a `pushAndPull` replication when using Conflict Free mode.
+If conflicts occur, you should configure a `pushAndPull` replication when using Conflict Free mode.
```

Comment:
There's only conflict free mode, so you should say a push replication will not be able to push documents if there are conflicts, and you to run a pull replication simultaneously.

I think just dropping `when Conflict Free mode` is sufficient.